tests/test_serialization.py

```diff
@@ -290,3 +290,14 @@ def test_pickle_none() -> None:
         fp.seek(0)
         cfg2 = pickle.load(fp)
         assert cfg == cfg2
+
+
+def test_pickle_flags_consistency() -> None:
+    cfg = DictConfig({"a": 0})
```

Comment:
Prefer OmegaConf.create to direct DictConfig usage.
```suggestion
    cfg = OmegaConf.create({"a": 0})
```